packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Providers/MrwSerializationTypeDefinition.cs

```diff
@@ -965,7 +1068,25 @@ private List<ParameterProvider> BuildSerializationConstructorParameters()
                     _rawDataField.Type));
             }
 
-            return constructorParameters;
+            return (constructorParameters, constructorInitializer);
+
+            static ConstructorSignature? GetBaseConstructor(TypeProvider? baseProvider)
```

Comment:
to be clear, make the input type to `MrwSerializationTypeDefinition`?

Reply:
well it does not have to be `MrwSerializationTypeProvider` because `TypeProvider` already have all the API I need. But this is a local method, it is fine to change.

Reply:
fixed